src/datetime.rs

```diff
@@ -168,7 +169,7 @@ impl crate::DateTime {
     ///
     /// let big = bson::DateTime::from_millis(i64::MAX);
     /// let chrono_big = big.to_chrono();
-    /// assert_eq!(chrono_big, chrono::MAX_DATETIME)
+    /// assert_eq!(chrono_big, chrono::DateTime::<chrono::Utc>::MAX_UTC)
```

Comment:
Did chrono drop `MAX_DATETIME`? Or was this something clippy was complaining about?

Reply:
Chrono deprecated `MAX_DATETIME`, yeah.